README.md

```diff
@@ -74,44 +51,39 @@ Kubernetes directly.
 **Configuration 3:**
 ![screenshot](./images/gke-deployment.png)
 
-A simple Python Flask web application (Prime-flask) was created for this
-demonstration which contains two endpoints:
+A simple Python [Flask](http://flask.pocoo.org/) web application (`Prime-flask`) was created for this demonstration which contains two endpoints:
 
 `http://[ip]:8080/factorial/` and
 
 `http://[ip]:8080/prime/`
 
-examples would look like:
+Examples of use would look like:
 
 ```console
-curl http://[ip]:8080/prime/10
+curl http://35.227.149.80:8080/prime/10
 The sum of all primes less than 10 is 17
 
-curl http://[ip]:8080/factorial/10
+curl http://35.227.149.80:8080/factorial/10
 The factorial of 10 is 3628800
 ```
 
-Also included is a utility to validate a successful deployment called
-`validate.sh`.
-
+Also included is a utility to validate a successful deployment.
 
 ## Prerequisites
+
 ### Tools
-In order to use the code in this demo you will need access to the following
-tools:
+In order to use the code in this demo you will need access to the following tools:
 
 * A bash, or bash-compatible, shell
 * Access to an existing Google Cloud project with the
-[Kubernetes Engine v1.10.0 or later](https://cloud.google.com/kubernetes-engine/docs/quickstart#before-you-begin)
-  service enabled
-* If you do not have a Google Cloud Platform account you can sign up [here](https://cloud.google.com)
-  and get 300 dollars of free credit on your new account.
+[Kubernetes Engine v1.10.0 or later](https://cloud.google.com/kubernetes-engine/docs/quickstart#before-you-begin) service enabled
+* If you do not have a Google Cloud Platform account you can sign up [here](https://cloud.google.com) and get 300 dollars of free credit on your new account.
 * [Google Cloud SDK (200.0.0 or later)](https://cloud.google.com/sdk/downloads)
 * [ApacheBench](https://httpd.apache.org/docs/2.4/programs/ab.html)
 * [HashiCorp Terraform (>= v0.11.7)](https://www.terraform.io/downloads.html)
 * [gcloud](https://cloud.google.com/sdk/gcloud/)
 * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
-
+* [Terraform](terraform.io)
```

Comment:
83 seems to address terraform, I think we can delete this line

Reply:
Yup